packages/@react-spectrum/actiongroup/src/ActionGroup.tsx

```diff
@@ -382,6 +382,9 @@ function ActionGroupMenu<T>({state, isDisabled, isEmphasized, staticColor, items
     let selectedItem = state.collection.getItem(state.selectionManager.firstSelectedKey);
     if (selectedItem) {
       summaryIcon = selectedItem.rendered;
+      if (typeof summaryIcon === 'string') {
```

Comment:
when is summary icon a string?

Reply:
i see now, in https://reactspectrum.blob.core.windows.net/reactspectrum/56295d9d6b9638af91591e3b0c8c608b49b3fb5f/storybook/index.html?path=/story/actiongroup--overflowmode-collapse-single-selection